Can not build on Windows

error: failed to run custom build command for `gettext-sys v0.19.8`
process didn't exit successfully: `N:\gxi\target\debug\build\gettext-sys-292f5e5391eb75fe\build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_DIR
cargo:rerun-if-env-changed=GETTEXT_DIR
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_BIN_DIR
cargo:rerun-if-env-changed=GETTEXT_BIN_DIR
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_LIB_DIR
cargo:rerun-if-env-changed=GETTEXT_LIB_DIR
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_INCLUDE_DIR
cargo:rerun-if-env-changed=GETTEXT_INCLUDE_DIR
OPT_LEVEL = Some("0")
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
CC_x86_64-pc-windows-msvc = None
CC_x86_64_pc_windows_msvc = None
HOST_CC = None
CC = None
CFLAGS_x86_64-pc-windows-msvc = None
CFLAGS_x86_64_pc_windows_msvc = None
HOST_CFLAGS = None
CFLAGS = None
CRATE_CC_NO_DEFAULTS = None
CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
DEBUG = Some("true")
running: "tar" "xJf" "C:\\Users\\vsmawoex\\CargoHome\\registry\\src\\github.com-1ecc6299db9ec823\\gettext-sys-0.19.8\\gettext-0.19.8.1.tar.xz" "--strip-components" "1" "--force-local"

--- stderr
tar: Option --force-local is not supported
Usage:
  List:    tar -tf <archive-filename>
  Extract: tar -xf <archive-filename>
  Create:  tar -cf <archive-filename> [filenames...]
  Help:    tar --help
thread 'main' panicked at '
command did not execute successfully, got: exit code: 1

build script failed, must exit now', C:\Users\vsmawoex\CargoHome\registry\src\github.com-1ecc6299db9ec823\gettext-sys-0.19.8\build.rs:251:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed

environment:
cargo 1.33.0 (f099fe94b 2019-02-12)
rustup 1.16.0 (beab5ac2b 2018-12-06)

The entire build log 

vsmawoex@VSMAWOEX  N\gxi   master ≣ ~2 -0 !  meson build
The Meson build system
Version: 0.50.0
Source dir: N:\gxi
Build dir: N:\gxi\build
Build type: native build
Project name: gxi
Project version: 0.5.4
Native Rust compiler: rustc (rustc 1.35.0-nightly)
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: D:\msys64\mingw64\bin\pkg-config.EXE (0.29.2)
Dependency glib-2.0 found: YES 2.58.2
Dependency gio-2.0 found: YES 2.58.2
Dependency cairo found: YES 1.16.0
Dependency pango found: YES 1.43.0
Dependency gtk+-3.0 found: YES 3.24.5
Program desktop-file-validate found: NO
Program appstream-util found: NO
Program cargo found: YES (C:\Users\vsmawoex\CargoHome\bin\cargo.EXE)
Program glib-compile-resources found: YES (D:\msys64\mingw64\bin\glib-compile-resources.EXE)
Program cargo-vendor found: YES (C:\Users\vsmawoex\CargoHome\bin\cargo-vendor.EXE)
Program scripts/cargo.sh found: YES (sh N:\gxi\scripts/cargo.sh)
Program scripts/grabber.sh found: YES (sh N:\gxi\scripts/grabber.sh)
meson.build:39: ?[1;33mWARNING:?[0m Custom target input '['']' can't be converted to File object(s).
This will become a hard error in the future.
Program ?[1mgit?[0m found: ?[1;32mYES?[0m (D:\Program Files\Git\bin\git.EXE)
meson.build:81: ?[1;33mWARNING:?[0m Custom target input '['']' can't be converted to File object(s).
This will become a hard error in the future.
Program ?[1mmeson_post_install.sh?[0m found: ?[1;32mYES?[0m (sh N:\gxi\meson_post_install.sh)
Build targets in project: ?[1m8?[0m
Found ninja-1.9.0 at 'D:\ninja\ninja.EXE'

 vsmawoex@VSMAWOEX  N\gxi   master ≣ ~2 -0 !  ninja -C build
ninja: Entering directory `build'
[0/4] Generating cargo-build-gxi with a custom command.
\tGXI Plugindir: \033[0;32mc:/lib/gxi/plugins\033[0m
\tGXI Localedir: \033[0;32mc:/share/locale\033[0m
\tGXI App-ID:    \033[0;32mcom.github.Cogitri.gxi\033[0m
\tGXI Version:   \033[0;32m0.5.4\033[0m
\tCrossbuild:    \033[0;32mfalse

   Compiling semver-parser v0.7.0
   Compiling pkg-config v0.3.14
   Compiling libc v0.2.50
   Compiling proc-macro2 v0.4.27
   Compiling winapi v0.3.6
   Compiling arrayvec v0.4.10
   Compiling rand_core v0.4.0
   Compiling build_const v0.2.1
   Compiling unicode-xid v0.1.0
   Compiling nodrop v0.1.13
   Compiling cc v1.0.31
   Compiling syn v0.15.29
   Compiling byteorder v1.3.1
   Compiling cfg-if v0.1.7
   Compiling lazy_static v1.3.0
   Compiling stable_deref_trait v1.1.1
   Compiling memchr v2.2.0
   Compiling scopeguard v0.3.3
   Compiling adler32 v1.0.3
   Compiling memoffset v0.2.1
   Compiling smallvec v0.6.9
   Compiling autocfg v0.1.2
   Compiling gio v0.6.0
   Compiling serde v1.0.89
   Compiling bitflags v1.0.4
   Compiling ryu v0.2.7
   Compiling crc32fast v1.2.0
   Compiling winapi-build v0.1.1
   Compiling cairo-rs v0.6.0
   Compiling regex v0.2.11
   Compiling pango v0.6.0
   Compiling quick-error v1.2.2
   Compiling ucd-util v0.1.3
   Compiling regex v1.1.0
   Compiling failure_derive v0.1.5
   Compiling fragile v0.3.0
   Compiling gdk-pixbuf v0.6.0
   Compiling safemem v0.3.0
   Compiling winapi v0.2.8
   Compiling xml-rs v0.8.0
   Compiling atk v0.6.0
   Compiling utf8-ranges v1.0.2
   Compiling itoa v0.4.3
   Compiling gdk v0.10.0
   Compiling rustc-demangle v0.1.13
   Compiling linked-hash-map v0.5.1
   Compiling lazycell v1.2.1
   Compiling fnv v1.0.6
   Compiling lazy_static v0.2.11
   Compiling bytecount v0.4.0
   Compiling gtk v0.6.0
   Compiling unicode-segmentation v1.2.1
   Compiling pangocairo v0.7.0
   Compiling gxi v0.5.4 (N:\gxi)
   Compiling xi-unicode v0.1.0 (N:\gxi\vendor\xi-editor\rust\unicode)
   Compiling semver v0.9.0
   Compiling rand_core v0.3.1
   Compiling crc v1.8.1
   Compiling glib-sys v0.8.0
   Compiling gobject-sys v0.8.0
   Compiling gio-sys v0.8.0
   Compiling gdk-pixbuf-sys v0.8.0
   Compiling cairo-sys-rs v0.8.0
   Compiling pango-sys v0.8.0
   Compiling gdk-sys v0.8.0
   Compiling atk-sys v0.8.0
   Compiling pangocairo-sys v0.9.0
   Compiling gtk-sys v0.8.0
   Compiling log v0.4.6
   Compiling filetime v0.2.4
   Compiling owning_ref v0.4.0
   Compiling crossbeam-utils v0.6.5
   Compiling thread_local v0.3.6
   Compiling miniz_oxide v0.2.1
   Compiling rand_chacha v0.1.1
   Compiling backtrace v0.3.14
   Compiling rand_pcg v0.1.2
   Compiling bincode v1.1.2
   Compiling rand v0.6.5
   Compiling kernel32-sys v0.2.2
   Compiling humantime v1.2.0
   Compiling regex-syntax v0.6.5
   Compiling regex-syntax v0.5.6
   Compiling line-wrap v0.1.1
   Compiling yaml-rust v0.4.3
   Compiling miniz_oxide_c_api v0.2.1
   Compiling miniz-sys v0.1.11
   Compiling onig_sys v69.1.0
   Compiling gettext-sys v0.19.8 (https://github.com/Koka/gettext-rs#6b0f498c)
   Compiling rand_hc v0.1.0
   Compiling rand_isaac v0.1.1
   Compiling rand_xorshift v0.1.1
   Compiling rustc_version v0.2.3
   Compiling lock_api v0.1.5
   Compiling crossbeam-channel v0.3.8
   Compiling num_cpus v1.10.0
error: failed to run custom build command for `gettext-sys v0.19.8 (https://github.com/Koka/gettext-rs#6b0f498c)`
process didn't exit successfully: `N:/gxi/build\release\build\gettext-sys-53ef043fdd094487\build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_SYSTEM
cargo:rerun-if-env-changed=GETTEXT_SYSTEM
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_DIR
cargo:rerun-if-env-changed=GETTEXT_DIR
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_BIN_DIR
cargo:rerun-if-env-changed=GETTEXT_BIN_DIR
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_LIB_DIR
cargo:rerun-if-env-changed=GETTEXT_LIB_DIR
cargo:rerun-if-env-changed=X86_64_PC_WINDOWS_MSVC_GETTEXT_INCLUDE_DIR
cargo:rerun-if-env-changed=GETTEXT_INCLUDE_DIR
OPT_LEVEL = Some("3")
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
CC_x86_64-pc-windows-msvc = None
CC_x86_64_pc_windows_msvc = None
HOST_CC = None
CC = None
CFLAGS_x86_64-pc-windows-msvc = None
CFLAGS_x86_64_pc_windows_msvc = None
HOST_CFLAGS = None
CFLAGS = None
CRATE_CC_NO_DEFAULTS = None
CARGO_CFG_TARGET_FEATURE = Some("fxsr,mmx,sse,sse2")
DEBUG = Some("false")
running: "tar" "xJf" "C:\\Users\\vsmawoex\\CargoHome\\git\\checkouts\\gettext-rs-715053e304988ca7\\6b0f498\\gettext-sys\\gettext-0.19.8.1.tar.xz" "--strip-components" "1" "--force-local"

--- stderr
tar: Option --force-local is not supported
Usage:
  List:    tar -tf <archive-filename>
  Extract: tar -xf <archive-filename>
  Create:  tar -cf <archive-filename> [filenames...]
  Help:    tar --help
thread 'main' panicked at '
command did not execute successfully, got: exit code: 1

build script failed, must exit now', C:\Users\vsmawoex\CargoHome\git\checkouts\gettext-rs-715053e304988ca7\6b0f498\gettext-sys\build.rs:251:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
[2/4] Generating com.github.Cogitri.gxi.desktop_data_merge with a custom command.
FAILED: gxi
"sh" "N:/gxi/scripts/cargo.sh" ".." "N:/gxi/build/." "gxi" "N:/gxi/build" "gxi" "false" "c:/lib/gxi/plugins" "c:/share/locale" "0.5.4"
ninja: build stopped: subcommand failed.

--- a/README.md
+++ b/README.md
@@ -75,3 +75,16 @@ Building and installing the git tracking package is identical, just replace all 
 
 Please consult the [Arch Wiki](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages)
 for more information regarding installing packages from the AUR.
+
+### Installation on Windows
+
+I'll try to have binary releases for Windows by gxi version 0.7.0. If you don't want to wait,
+or want to use a debug build for developing follow the following instructions:
+
+0) Install Rust by visiting https://rustup.rs. After running the exe press `2` (right after you see the terminal of rustup-init.exe) to customize the settings and enter `x86_64-pc-windows-gnu` as default triplet (notice the `gnu` instead of `msvc`)
+1) Go to https://www.msys2.org/ and download the appropriate installer (usually x86_64)
+2) Go into your start menu and open the MSYS terminal
+3) Enter `pacman -S mingw-w64-x86_64-toolchain mingw-w64-x86_64-gtk3 git` in the terminal
+4) Open the `MinGW64` terminal from your start menu. Do `echo 'PATH="/c/Users/${USER}/.cargo/bin:${PATH}"' >> .bash_profile`
+5) Reload the just made changes with `source .bash_profile`. Then clone gxi: `git clone https://github.com/Cogitri/gxi`.
+6) `cd gxi && cargo run` <- This should produce a debug build for you and run it.

--- a/src/ui/gxi.glade
+++ b/src/ui/gxi.glade
@@ -254,7 +254,6 @@
     <property name="default_width">1150</property>
     <property name="default_height">750</property>
     <property name="icon_name">text-x-generic</property>
-    <property name="startup_id">com.github.Cogitri.gxi</property>
     <child type="titlebar">
       <object class="GtkHeaderBar" id="header_bar">
         <property name="visible">True</property>

